src/gc/impl/conservative/gc.d

```diff
@@ -1596,7 +1596,7 @@ struct Gcx
 
     private @property bool lowMem() const nothrow
     {
-        return isLowOnMem(mappedPages * PAGESIZE);
+        return isLowOnMem(cast(size_t)mappedPages * PAGESIZE);
```

Comment:
I wonder if it would be better to instead change the type of the `PAGESIZE` enum to `size_t`. That way any operation where one of the operands is `PAGESIZE` would promote the other operand as necessary.